app/controllers/games_controller.rb

```diff
@@ -0,0 +1,20 @@
+class GamesController < ApplicationController
+
+
+	def new
+
+	end
+
+
+	def create
```

Comment:
These actions should be hooked up and functioning. You don't need to display anything but some placeholder text in the views but the database models should be persisted.